lib/config.js

```diff
@@ -14,6 +14,14 @@ const conf = convict({
       default: 1
     }
   },
+  authServer: {
+    url: {
+      doc: 'URL of fxa-auth-server',
+      env: 'AUTH_SERVER_URL',
+      format: 'url',
+      default: 'http://127.0.0.1:9000/v1'
+    }
+  },
```

Comment:
As for position in the config file, I've been trying to keep it alphabetically sorted.